src/wiki/getting-started/installing-java.md

```diff
@@ -20,6 +20,55 @@ If you aren't sure which version you will need, **it is recommended to just inst
 
 You can also use Oracle Java, but openjdk is better here. Also to download Oracle Java 9+ you need an Oracle account which is not a good thing.
 
+# Java on linux
+
+On Linux, it's recommended to use your package manager for installing Java.
+
+## Fedora
+
+On the COPR package all required java versions should be installed, but this is the command to install it:
+
+```
+sudo dnf install java-1.8.0-openjdk java-17-openjdk
+```
+
+## Void Linux
+
+```
+sudo xbps-install openjdk17-jre openjdk8-jre
+```
+
+## Arch Linux
+
+```
+sudo pacman -S jre17-openjdk jre8-openjdk
+```
+
+## Ubuntu
+
+```
+sudo apt install openjdk-17-jre openjdk-8-jre
+```
+
+## Debian
+
+```
+sudo apt install openjdk-17-jre 
+```
+
+Java 8 is not available from Debian repos for unknown reasons, but you can [get a deb from Azul](https://www.azul.com/downloads/?version=java-8-lts&os=debian&package=jre&show-old-builds=true)
```

Comment:
Looks like it actually is available, but only on oldoldstable and sid branches:
![image](https://user-images.githubusercontent.com/31766876/172230400-b9304c98-5993-4afd-a6a2-deb8b06b6466.png)

Reply:
https://packages.debian.org/search?keywords=openjdk-8-jre&searchon=names&suite=all&section=all

Reply:
well, you shouldn't use oldoldstable (PolyMC won't work on it anyways) and most people aren't going to use those versions